src/DotVVM.Framework/Resources/Scripts/spa/spa.ts

```diff
@@ -8,27 +8,29 @@ import { DotvvmPostbackError } from '../shared-classes';
 export const isSpaReady = ko.observable(false);
 
 export function init(): void {
-    const spaPlaceHolder = getSpaPlaceHolder();
-    if (spaPlaceHolder == null) {
-        throw new Error("The SpaContentPlaceHolder control was not found!");
+    const spaPlaceHolders = getSpaPlaceHolders();
+    if (spaPlaceHolders == null || spaPlaceHolders.length == 0) {
```

Comment:
`spaPlaceHolders` will never be null, see https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementsByName